src/pages/home/report/ReactionList/BaseReactionList.tsx

```diff
@@ -1,64 +1,64 @@
 /* eslint-disable rulesdir/onyx-props-must-have-default */
 import Str from 'expensify-common/lib/str';
-import PropTypes from 'prop-types';
 import React from 'react';
 import {FlatList} from 'react-native';
+import type {FlatListProps} from 'react-native';
 import OptionRow from '@components/OptionRow';
-import participantPropTypes from '@components/participantPropTypes';
-import withWindowDimensions from '@components/withWindowDimensions';
 import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
 import Navigation from '@libs/Navigation/Navigation';
 import * as UserUtils from '@libs/UserUtils';
 import variables from '@styles/variables';
 import CONST from '@src/CONST';
 import ROUTES from '@src/ROUTES';
+import type {PersonalDetails} from '@src/types/onyx';
 import HeaderReactionList from './HeaderReactionList';
-import reactionPropTypes from './reactionPropTypes';
+import type ReactionListProps from './types';
 
-const propTypes = {
+type BaseReactionListProps = ReactionListProps & {
     /**
      *  Array of personal detail objects
      */
-    users: PropTypes.arrayOf(participantPropTypes).isRequired,
+    users: PersonalDetails[];
 
     /**
      * Returns true if the current account has reacted to the report action (with the given skin tone).
      */
-    hasUserReacted: PropTypes.bool,
+    hasUserReacted: boolean;
 
-    ...reactionPropTypes,
-};
-
-const defaultProps = {
-    hasUserReacted: false,
+    /**
+     * Returns true if the reaction list is visible
+     */
+    isVisible: boolean;
```

Comment:
Should it be a required prop?

Reply:
Absolutely not. If not passed, it basically means it's hidden. Will make it optional.